ReText/window.py

```diff
@@ -39,7 +39,7 @@
 from PyQt5.QtWidgets import QAction, QActionGroup, QApplication, QCheckBox, \
  QComboBox, QDesktopWidget, QDialog, QFileDialog, QFontDialog, QInputDialog, \
  QLineEdit, QMainWindow, QMenuBar, QMessageBox, QSplitter, QTabWidget, \
- QTextBrowser, QTextEdit, QToolBar
+ QTextBrowser, QTextEdit, QToolBar, QShortcut
```

Comment:
Please keep the list sorted. `QShortcut` should go between `QMessageBox` and `QSplitter`.